data/testimonials.yml

```diff
@@ -3,7 +3,7 @@ items:
     text: No company can realize the IoT on its own… Within the Eclipse Community, through the contribution of many IoT developers, tools and standards are created on an open platform that many companies can benefit from for their IoT applications.
     author: Stefan Ferber
     job_title: CEO
-    company: Bosch Software Innovations
+    company: Bosch .IO GmbH
```

Comment:
I've ask Zahra to confirm this: https://github.com/EclipseFdn/iot.eclipse.org/issues/589#issuecomment-763711689